blosc2/c2array.py

```diff
@@ -21,48 +21,83 @@
 C2SUB_URLBASE_ENVVAR = 'BLOSC_C2URLBASE'
 """Environment variable with a default Caterva2 subscriber URL base."""
 
+C2SUB_USERNAME_ENVVAR = 'BLOSC_C2USERNAME'
```

Comment:
To me, I'd have a better grasp at the code if I see strings like `'BLOSC_C2USERNAME'` as-is, instead of `C2SUB_USERNAME_ENVVAR` but not a show-stopper at all.